gensim/utils.py

```diff
@@ -1164,11 +1164,12 @@ def check_output(stdout=subprocess.PIPE, *popenargs, **kwargs):
     Added extra KeyboardInterrupt handling
     """
     try:
+        cmd = kwargs.get("args")
+        logger.info("COMMAND: %s", cmd)
```

Comment:
I think if some specific information needs to be logged for a command, then the log statement can be specified in the code there and this will just make sure to log the command that has to be run for debugging purposes. WDYT?

Or should a parameter for a log string be provided so that the command specific info is also logged from check_output